test/widgets/message_list_test.dart

```diff
@@ -648,6 +645,31 @@ void main() {
       check(isMarkAsReadButtonVisible(tester)).isFalse();
     });
 
+    testWidgets('messages don\'t shift position', (WidgetTester tester) async {
```

Comment:
nit: make this a double-quoted string, to look a bit cleaner with the embedded apostrophe

Reply:
Fixed.